src/main/java/net/dv8tion/jda/internal/requests/restaction/pagination/AuditLogPaginationActionImpl.java

```diff
@@ -158,4 +158,10 @@ protected void handleSuccess(Response response, Request<List<AuditLogEntry>> req
 
         request.onSuccess(list);
     }
+
+    @Override
+    protected long getKey(AuditLogEntry it)
```

Comment:
Will `getKey` ever be anything other than a Snowflake's ID?
Can we just build a built-in reliant on `Snowflake#getIdLong`?